ProjectApplication/project_core/filters.py

```diff
@@ -6,22 +6,24 @@
 
 
 class ProjectFilterSet(FilterSet):
-    key_title = CharFilter(method='filter_key_title', label="Key or Title")
-    keywords = CharFilter(method='filter_keywords')
+    key_title = CharFilter(method='filter_any', label="Key, Title, Keyword, PI, laysummary or organisation")
     geographical_areas = CharFilter(method='filter_geographical_areas')
     funding_instrument = CharFilter(method='filter_funding_instrument')
     start_date = CharFilter(method='filter_start_date')
 
-    def filter_key_title(self, queryset, name, value):
+    def filter_any(self, queryset, name, value):
         search_query = Q(
             Q(key__icontains=value) |
-            Q(title__icontains=value)
+            Q(title__icontains=value) |
+            Q(keywords__name__icontains=value) |
+            Q(principal_investigator__person__first_name__icontains=value) |
+            Q(principal_investigator__person__surname__icontains=value) |
```

Comment:
This doesn't really do what I think a user would expect.

If a user searches for "carles pina" (firstname and surname) I would expect to find it. But since "first_name" doesn't contain "carles pina" and surname doesn't contain it either it will not be found.